src/services/meal.service.ts

```diff
@@ -9,49 +9,90 @@ import {
 import config from "../config.json";
 
 export const getMealOptions = (): Promise<MealOptions> => {
-  return axios.get(`${config.apiUrl}/meal-options`).then((res) => res.data);
+  const storedData = JSON.parse(localStorage.getItem("userData"));
+  return axios
+    .get(`${config.apiUrl}/meal-options`, {
+      headers: {
```

Comment:
If you decided to pass the token with each request, i would suggest extracting the logic that fetches the token and creates the header, something like: `createAuthotizationHeader`